### PR TITLE
Fix enums, remove annexes

### PR DESCRIPTION
**Pull request by karelbilek**
Apply these PRs from gunk docgen:

* https://github.com/gunk/gunk/pull/300
* https://github.com/gunk/gunk/pull/302
* https://github.com/gunk/gunk/pull/296
* https://github.com/gunk/gunk/pull/294

### CODE REVIEW

**Comment by karelbilek** [comment]
asking @favadi for review (finished now)

**Review by favadi — APPROVED** [review]

**Comment by mvdan** [comment]
@ghazninattarshah @nkristianto @jwangsadinata any last thoughts before we merge? if not, let's just merge tomorrow

File: v1/account/all.md
Comment (ghazninattarshah) [inline v1/account/all.md:705]: Some fields have period at the end some fields aren't, let it be consistent.

File: v1/account/all.md
Comment (ghazninattarshah) [inline v1/account/all.md:56]: Do we need the description ending with `period` `.`, can we remove it inside the table?

**Review by ghazninattarshah — APPROVED** [review]
LGTM very minor comments.

File: v1/account/all.md
Comment (karelbilek) [reply v1/account/all.md:56]: I don't want to do it programatically, we should delete the dots from comments if we don't want them, rather than do some trimming in gunk

(I agree with you, just saying where to do it)

File: v1/account/all.md
Comment (favadi) [reply v1/account/all.md:705]: It doesn't seem to be related to this PR, should be fixed in a separate PR later.

File: v1/account/all.md
Comment (favadi) [reply v1/account/all.md:56]: It should not be gunk's work to enforce styling. The trailing . should be removed by users themselves.

Comment (ghazninattarshah) [reply v1/account/all.md:56]: I thought it was gunk added it, ignore this comment.

**Comment by karelbilek** [comment]
please merge, I don't have merge rights here (?)

**Pull request merged**
